src/app/pages/SearchPage/SearchPageDucks.ts

```diff
@@ -94,6 +96,7 @@ type StoreValue = { [REDUCER_KEY]: SearchPageState }
 export const getSort = ({ [REDUCER_KEY]: { sort } }: StoreValue) => sort
 export const getPage = ({ [REDUCER_KEY]: { page } }: StoreValue) => page
 
+export const getQuery = ({ [REDUCER_KEY]: { query } }: StoreValue) => query?.toString()
```

Comment:
There is no need to use optional chaining here as the `SearchPageState` guarantees that `query` is always present and of type `string`.  If this is not the case the code that sets this value should be changed.

```suggestion
export const getQuery = ({ [REDUCER_KEY]: { query } }: StoreValue) => query
```